.storybook/middleware.js

```diff
@@ -0,0 +1,10 @@
+const { createProxyMiddleware  } = require("http-proxy-middleware");
+module.exports = function expressMiddleware(router) {
+  router.use(
+    "/config",
+    createProxyMiddleware({
+      target: "https://incident-commander.canary.lab.flanksource.com",
+      changeOrigin: true
+    })
+  );
+};
```

Comment:
Let's not use external server for story book tests. You could mock request response for individual story.